gateway/server.go

```diff
@@ -207,7 +207,22 @@ func main() {
 	}
 
 	metricsHandler := metrics.PrometheusHandler()
-	r.Handle("/metrics", metricsHandler)
+
+	//Create a new server to serve /metrics endpoint
+	go func() {
```

Comment:
Let's do this in a separate method at least?

Reply:
I have put this in a separate method and added `/healthz` endpoint.